main/src/main/java/org/mobilitydata/gtfsvalidator/util/HtmlOutputUtil.java

```diff
@@ -0,0 +1,199 @@
+  private static final String SEVERITY = "severity";
+  private static final String TOTAL_NOTICES = "totalNotices";
+  private static final String SAMPLE_NOTICES = "sampleNotices";
+
+  public static String outputBuilder(
+      JsonObject noticesJson, String jqueryDependency, String styleCss, String scriptJs) {
+    StringBuilder notices = new StringBuilder();
+    HashMap<String, Integer> noticesQty =
+        new HashMap<>() {
+          {
+            put(ERROR, 0);
+            put(WARNING, 0);
+            put(INFO, 0);
+          }
+        };
+    JsonArray noticesArray = noticesJson.getAsJsonArray(NOTICES);
+    for (JsonElement notice : noticesArray) {
+      JsonObject noticeJson = notice.getAsJsonObject();
+      String noticeCode = noticeJson.get(CODE).getAsString();
+      String noticeSeverity = noticeJson.get(SEVERITY).getAsString();
+      int totalNotices = noticeJson.get(TOTAL_NOTICES).getAsInt();
+      noticesQty.put(noticeSeverity, noticesQty.get(noticeSeverity) + totalNotices);
+      JsonArray sampleNotices = noticeJson.getAsJsonArray(SAMPLE_NOTICES);
+      JsonElement sampleNotice = sampleNotices.get(sampleNotices.size() - 1);
+      JsonObject sampleJson = sampleNotice.getAsJsonObject();
+      ArrayList<String> noticeFields = new ArrayList<>();
+      Set<Map.Entry<String, JsonElement>> sampleEntries = sampleJson.entrySet();
+      for (Map.Entry<String, JsonElement> entry : sampleEntries) {
+        noticeFields.add(entry.getKey());
+      }
+      String sampleColumns = noticeColumnsBuilder(noticeFields);
+      String sampleRows = noticeRowsBuilder(noticeFields, sampleNotices);
+      notices.append(
+          noticeFrame(noticeCode, noticeSeverity, totalNotices, sampleColumns, sampleRows));
+    }
+    String outputHeader =
+        outputHeader(
+            (noticesQty.get(ERROR) + noticesQty.get(WARNING) + noticesQty.get(INFO)),
+            noticesQty.get(ERROR),
+            noticesQty.get(WARNING),
+            noticesQty.get(INFO));
+    return outputFrame(
+        jqueryDependency, styleCss, scriptJs, outputHeader, notices.toString(), outputFooter());
+  }
+
+  public static String resourceLoader(String resourcePath) {
+    String content = "";
+    try (InputStream stream =
+        HtmlOutputUtil.class.getClassLoader().getResourceAsStream(resourcePath)) {
+      Scanner scanner = new Scanner(stream).useDelimiter("\\A");
+      content = scanner.hasNext() ? scanner.next() : "";
+    } catch (IOException e) {
+      System.out.println("The resource was not found at \"" + resourcePath + "\".");
+    }
+    return content;
+  }
+
+  protected static String noticeColumnsBuilder(ArrayList<String> noticeFields) {
+    StringBuilder columns = new StringBuilder("                ");
+    for (String field : noticeFields) {
+      columns.append("<th>").append(field).append("</th>");
+    }
+    columns.append("\n");
+    return columns.toString();
+  }
+
+  protected static String noticeRowsBuilder(
```

Comment:
Maybe it's just me, but I find it hard to read some of the data that's dumped into tables now, like this:

![image](https://user-images.githubusercontent.com/928045/166064179-cddb8b97-b3a6-46ea-bfb7-7d9909cc73b7.png)

Creating human-readable sentences would obviously be an improvement - is that outside of the scope of this PR?

I was thinking similar to what we have on the GTFS Realtime Validator:

![image](https://user-images.githubusercontent.com/928045/166064320-f13840de-a562-48d9-a58a-13e7174add61.png)